processor/src/main/java/com/linecorp/decaton/processor/LoggingContext.java

```diff
@@ -43,8 +45,10 @@ public class LoggingContext implements AutoCloseable {
     public LoggingContext(boolean enabled, String subscriptionId, TaskRequest request, TaskMetadata metadata) {
         this.enabled = enabled;
         if (enabled) {
+            final String taskKey = request.key() == null ? "" : new String(request.key(), StandardCharsets.UTF_8);
```

Comment:
null-key should be string "null" rather than empty string to align to current `String.valueOf` semantics.

Reply:
fixed https://github.com/line/decaton/pull/157/commits/b9eac225ae18733e87306098fd6cce11cc5fe6cd